js/components/CategoriesItem.jsx

```diff
@@ -7,11 +7,50 @@ var ScoreBar = require('./ScoreBar.jsx');
 var Link = Router.Link;
 
 var CategoriesItem = React.createClass({
+  getInitialState: function() {
```

Comment:
Moved logic to get investors up to the CategoriesItem because it was weird to have it so low